packages/cursorless-engine/src/processTargets/modifiers/PositionStage.ts

```diff
@@ -1,11 +1,46 @@
+import { EndOfModifier, Range, StartOfModifier } from "@cursorless/common";
 import { Target } from "../../typings/target.types";
-import { PositionModifier } from "@cursorless/common";
 import { ModifierStage } from "../PipelineStages.types";
+import {
+  CommonTargetParameters,
+  PlainTarget,
+  RawSelectionTarget,
+} from "../targets";
 
-export default class PositionStage implements ModifierStage {
-  constructor(private modifier: PositionModifier) {}
-
+abstract class StartEndOfStage implements ModifierStage {
   run(target: Target): Target[] {
-    return [target.toPositionTarget(this.modifier.position)];
+    const parameters: CommonTargetParameters = {
+      editor: target.editor,
+      isReversed: target.isReversed,
+      contentRange: this.constructContentRange(target.contentRange),
+    };
+
+    return [
+      target.isRaw
+        ? new RawSelectionTarget(parameters)
+        : new PlainTarget({ ...parameters, isToken: false }),
```

Comment:
It is so that we don't lose raw/just. `end of just air` vs `end of air`.
This test for example fails if I remove that logic:
https://github.com/cursorless-dev/cursorless/blob/9c3366a2e4f71d7cb96e69fb3a342dd18618e185/packages/cursorless-vscode-e2e/src/suite/fixtures/recorded/positions/bringArgMadeAndGustToEndOfJustThis.yml

Reply:
maybe try `new RawSelectionTarget(parameters)` always and see if tests pass

Reply:
worked